packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -164,18 +161,32 @@ export async function processContent(mode: Mode, concurrently = true) {
 		// Clean up any failed snapshots that might have been written out in previous test run.
 		cleanFailedSnapshots(folder);
 
-		// SnapFreq is the most interesting options to tweak
-		// On one hand we want to generate snapshots often, ideally every 50 ops
-		// This allows us to exercise more cases and increases chances of finding bugs.
-		// At the same time that generates more files in repository, and adds to the size of it
-		// Thus using two passes:
-		// 1) Stress test - testing eventual consistency only
-		// 2) Testing backward compat - only testing snapshots at every 1000 ops
-		const snapFreq = mode === Mode.Stress ? 50 : 1000;
+		// SnapFreq is the most interesting options to tweak. On one hand we want to generate snapshots often, ideally
```

Comment:
Nit: personal preference here, I'd lean towards the sentence-per-line approach (also used in markdown files) so that fixing typos or changing one sentence in a paragraph is easy to spot in a diff; if a change causes lines to start becoming too long we then need to break them differently, that could affect the lines below, etc, and it gets a bit more cumbersome to review.

Reply:
I don't have a preference. I just don't like large block of comments that can be shortened :-).
Changed to be per line comment.